FluentUI/build.gradle

```diff
@@ -15,6 +15,10 @@ android {
         consumerProguardFiles 'consumer-rules.pro'
         vectorDrawables.useSupportLibrary = true
     }
+    
+    lintOptions {
```

Comment:
was it already there originally ?

Reply:
Its their in master branch and full build gives error without this